Embedded Link -> Link (#1031)
codatio/codat-docs: docs/auth-flow/overview.md

---

> title: "Authorization flow"

Reviewer: I think we should fast follow this release with a video showing off link etc. Let's discuss

> description: "Understand Codat's authorization journey and its implementation"

Reviewer: I think the subtitle should contextualise that the auth flow is the SMB's experience and what it achieves, maybe something like
```suggestion
description: "Get access to your SMB customers' data with Link SDK"
```

---

> Authorization is a key part of any solution built with Codat. Every company must authorize access to their data before you can use that data. A frictionless and reassuring auth flow is essential for receiving the authorization from your SMB customers.

Reviewer: ```suggestion
Authorization is a key part of any solution built with Codat. Each of your SMB customers must authorize access to their data before you can use that data. A frictionless and reassuring auth flow is essential for receiving the authorization from your SMB customers.
```

It was presumably Company with a capital C before as it meant the data type Company, rather than the above

To achieve this, use our [Link SDK component](/auth-flow/authorize-embedded-link) and embed our Link auth flow in your application. 

![](/img/auth-flow/embedded-link-selection.png)

## What is Link?

> Link is our pre-built and white-labelled authorization journey, designed using our extensive build experience and authorization best practices to maximize conversion. 

Reviewer: ```suggestion
Link is our pre-built and white-labellable authorization journey, designed using our extensive build experience and authorization best practices to maximize conversion. 
```
I guess it should be like this, it hasn't yet been white labelled, it *can* be white labelled

> Embed this best-in-class auth flow into your user journey with our [Link SDK component](/auth-flow/authorize-embedded-link) and enjoy an **89% auth flow completion rate**.

Author: ```suggestion
Embed this best-in-class auth flow into your user journey with our [Link SDK component](/auth-flow/authorize-embedded-link) and enjoy an average **auth flow completion rate of 89%**.
```

> With our [rich examples](/auth-flow/authorize-embedded-link#get-started) and an <a href="https://links.codat.io/client/873ff19e-6fe0-47b0-a4e1-e19f344c78f6?user=8ee6c557-949c-40a8-b31d-e1fa02ef7fbc" target="_blank">interactive demo</a>, we have made it easy for you to get started. You can have the flow up and running in your front-end code with just a few lines of code.

Reviewer: ```suggestion
With our [rich examples](/auth-flow/authorize-embedded-link#get-started) and an <a href="https://links.codat.io/client/873ff19e-6fe0-47b0-a4e1-e19f344c78f6?user=8ee6c557-949c-40a8-b31d-e1fa02ef7fbc" target="_blank">interactive demo</a>, we've made it easy for you to get started. You can have the flow up and running in your front-end code with just a few lines of code.
```
Style-wise what should our position be on abbreviations?

Author: following the Google style guide, negation contractions (such as isn't instead of is not), are a good thing to do, but all the other contractions (such as we've) is not a good thing to do

      <h3>Control</h3>
    </div>
    <p>
      Codat gives you control of the auth flow so you can serve your use case, provide an experience aligned with your brand, and give your customers an intuitive user experience.
    </p>
  </li>
</ul>

> ## Compatibility

Reviewer: When I see the word compatibility I also think of stack compatibility - what tech do I need to use to use this

Reviewer: Should we mention languages and geographies at all? Maybe no

Author: is there anything to call out re: languages? is there a technology scenario where someone cannot use Link SDK?

Reviewer: Supported products and integrations

> ## Compatibility

Author: ```suggestion
## Supported products and integrations
```

> Link is compatible with our entire [product range](/using-the-api/overview#apis) except [Sync for Commerce](/commerce/overview). To set up your Sync for Commerce authorization flow, follow the instructions in our [dedicated documentation](/commerce/overview).

Reviewer: Good call out
